lib/runBlock.js

```diff
@@ -177,7 +177,7 @@ module.exports = function (opts, cb) {
           error: err
```

Comment:
Is this field expected to be an `Error` object or text?

Reply:
I just stayed in the semantics of the change with the new ``Error`` object introduced before.

Tested in CL, this works if an ``Error`` object is provided, but it uses just the error description from the first error and not the string. Is this acceptable? I'm not so deep into error handling...